homeassistant/components/netatmo/climate.py

```diff
@@ -220,20 +210,18 @@ def __init__(
         self._operation_list = [HVAC_MODE_AUTO, HVAC_MODE_HEAT]
         self._support_flags = SUPPORT_FLAGS
```

Comment:
Do this and you can remove the property:

```suggestion
        self._attr_supported_features = SUPPORT_FLAGS
```

Since it's a constant, can even define it above the `__init__`. Same with all other properties.